lisa/analysis/tasks.py

```diff
@@ -498,6 +527,33 @@ def plotter(axis, local_fig):
 
         return self.do_plot(plotter, filepath, axis, height=8)
 
+    @df_tasks_total_residency.used_events
+    def plot_tasks_total_residency(self, tasks, ascending=False, filepath=None, axis=None):
```

Comment:
What do you think of having a default `tasks=None` and this plots all of the tasks in the trace?

Reply:
Doable... although it produces a huge plot :open_mouth: 
Anyway, let's add it... ;)

Reply:
Maybe you can only plot e.g. the top 100 longest-running tasks by default to not DDoS-by-matplotlib the user.

Reply:
Which means we should add another parameter to defined how many tasks we want, and keep all tasks by default... ok... let's do that too ;)